File: Python Excel Examples/WorksheetsApiDemo/auto_fit_rows_columns/autofitRows.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "AutofitRows.xlsx"
password = ""
storage = ""
folder = "ExcelDocument"
sheet_name = "Sheet1"
start_Row = 17
end_Row = 23
only_auto = "true"
api.autofit_rows(name, sheet_name, start_Row, end_Row, only_auto=only_auto, folder=folder, storage=storage)

File: JavaScript Excel Examples/WorksheetsApiDemo/AutofitRowsColumns/autofitRows.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.WorksheetsApi(apiClient);
    
    var name = "autofitRows.xlsx";
    var opts = {onlyAuto : true ,//The default value of onlyAuto is false
                folder: "input"}; 
    var sheetName = "Sheet1";
    var startRow = 17;
    var endRow = 23;
    instance.autofitRows(name, sheetName, startRow, endRow, opts,
        function (error, data, response) {
        if (error) throw error;
    });
})();